packages/common/src/vscode-webui-bridge/webview.ts

```diff
@@ -300,6 +300,11 @@ export interface VSCodeHostApi {
 
   readModelList(): Promise<ThreadSignalSerialization<DisplayModel[]>>;
```

Comment:
no need to add an api, you can simply let readModelList return both a signal and a function (the function will trigger the reload.
readModelList() : Promise<{signal: ..., refresh: () => Promise<void>}>;